Src/EmbeddedScripts.CSharp.MonoEvaluator/Hosting/HostFields.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+
+namespace EmbeddedScripts.CSharp.MonoEvaluator.Hosting
+{
+    public static class HostFields
+    {
+        private static readonly Dictionary<int, Dictionary<string, object>> Fields = 
```

Comment:
It's better to use ConcurrentDictionary or use explicit lock in case of multiple evaluators will access it from different threads